requirements/local.txt

```diff
@@ -18,3 +18,10 @@ pytest-sugar==0.8.0
 
 # Own stuff
 django-rosetta==0.7.12
+
+
+# REST-Framework
+
+djangorestframework==3.3.0
+markdown==2.1.0
+django-filter==1.0.1
```

Comment:
Sollten die ganzen Requirements nicht in `base.txt` rein, damit sie auch in der Production installiert werden? So werden sie nur lokal installiert